Lib/pathlib.py

```diff
@@ -194,25 +194,24 @@ def resolve(self, path, strict=False):
         s = str(path)
         if not s:
             return os.getcwd()
+        if _getfinalpathname is None:
+            return None  # Means fallback on absolute
```

Comment:
Is it still possible for `_getfinalpathname` to be None? `GetFinalPathNameByHandle` was introduced on Windows Vista, and we don't support earlier systems. @zooba What do you think?

Reply:
No, it definitely exists on Windows (but we keep the fallback so that non-Windows users can `import ntpath`)

Reply:
Ok, but this is `pathlib` and `WindowsPath` isn't supposed to work on non-Windows systems :-)